desafios/desafio10.js

```diff
@@ -0,0 +1,5 @@
+use commerce;
+db.produtos.updateOne(
+  { "nome": { $eq: "Quarteirão com Queijo" } },
```

Comment:
Mano, não precisa do operador $eq! Ele só vai te fazer gastar uns char a mais. Se você só passar a string "Quarteirão com queijo", seu problema já é resolvido.
```suggestion
  { "nome": "Quarteirão com Queijo"  },
```

Reply:
Hmmm, boa lindao, vlw pelo toque